src/appmain.ts

```diff
@@ -246,14 +253,11 @@ export class ReplayWebApp extends LitElement {
   }
 
   renderNavBrand() {
-    return html` <img
-        style="height: 1.25rem; user-drag: none; user-select: none; -moz-user-select: none;"
-        src="data:image/svg+xml,${encodeURIComponent(
-          brandLockupColor as string,
-        )}"
-        alt="ReplayWeb.page"
-        draggable="false"
-      />
+    return html` <div class="text-left">
+        <svg width="9.5rem" height="1.25rem">
```

Comment:
This results in double SVG tags, should probably be a div?  Only have to set the height I think?  Something like this maybe?  Might also be nice to apply `wr-logo-item` to the div instead of the link so that the image isn't draggable but the link is, allowing users to drag a shortcut to their desktop — something I expect nobody to really do, but no reason to disable it?
```html
<div role="img" aria-label="ReplayWeb.page" style="height: 1.25rem;">
```

Reply:
The nested SVG tags are needed, this is what fa-icon does as well.